### PR TITLE
[nnpackage_run] Support multiple input for load:raw option

### PR DESCRIPTION
**Pull request by glistening**
nnpackage_run can accept multiple inputs.

--load:raw in

will get input raw data from in.0, in.1, in.2,..., in.{N-1}
where N is number of inputs, determined by nnfw info query api.

Signed-off-by: Sanggyu Lee <sg5.lee@samsung.com>

### Related Issue
#9012

### For Reviewers

```
$ tree add.q8
add.q8
├── add.q8.circle
├── add.q8.expected.0
├── add.q8.in.0
├── add.q8.in.1
├── add.q8.tvn
└── metadata
    └── MANIFEST

$ BACKENDS="trix;cpu" ./Product/x86_64-linux.debug/out/bin/nnpackage_run \
--load:raw add.q8/add.q8.in \
--dump:raw add.q8/add.q8.out \
add.q8

( ... )

add.q8/add.q8.out.0 is generated.
===================================
MODEL_LOAD   takes 0.959 ms
PREPARE      takes 6.088 ms
EXECUTE      takes 331.659 ms
- MEAN     :  331.659 ms
- MAX      :  331.659 ms
- MIN      :  331.659 ms
- GEOMEAN  :  331.659 ms
===================================

$ diff add.q8/add.q8.{expected,out}.0 
$ echo $?
0
```

### CODE REVIEW

**Review by chunseoklee — APPROVED** [review]

**Comment by hseok-oh** [comment]
Then do we need to update test nnpackage's input data file name for trix backend?

**Comment by glistening** [comment]
> Then do we need to update test nnpackage's input data file name for trix backend?

Yes, if you would like to use inferer's dumped input as golden input, you need to rename before you feeding this input file to `nnpackage_run`.

File: tests/tools/nnpackage_run/src/rawformatter.cc [hunk 1]
```diff
@@ -30,13 +30,10 @@ void RawFormatter::loadInputs(const std::string &filename, std::vector<Allocatio
...
-  // Option 2. Get prefix --load:raw out
-  //           Internally access out.0, out.1, out.2, ... out.{N} where N is determined by api.
-  if (num_inputs != 1)
-  {
-    throw std::runtime_error("Only 1 input is supported for raw input");
-  }
+  // Option 1: Get comman-separated input file list like --load:raw a,b,c
+  // Option 2: Get prefix --load:raw in
+  //           Internally access in.0, in.1, in.2, ... in.{N-1} where N is determined by nnfw info
+  //           query api.
```
Comment (glistening) [inline tests/tools/nnpackage_run/src/rawformatter.cc:36]: I thought one more option (= Option3) before this PR.
```
Option 3: Get template --load:raw in.#.raw
                Get in.0.raw, in.1.raw, ... in.{N-1}.raw.
```
But I am not sure it is better to use for user. Also, it requires more implementation.

@hseok-oh Do you like this option? Or do you have any idea or suggestion?

Comment (hseok-oh) [reply tests/tools/nnpackage_run/src/rawformatter.cc:36]: No. Let's update package's raw data file name.

**Review by hseok-oh — APPROVED** [review]

**Pull request merged**